gdk/x11/gdkdisplay-x11.c

```diff
@@ -430,6 +430,39 @@ get_event_xwindow (XEvent             *xevent)
   return xwindow;
 }
 
+#ifdef HAVE_XCOMPOSITE
```

Comment:
`HAVE_XCOMPOSITE` is a guard for using the XComposite API, not if the server has the compositing extension. You don't need this here.